README.md

```diff
@@ -64,9 +64,11 @@ present, `mysqld` is executed and will run as PID 1. You can stop the
 detached container by running `docker stop <CONTAINER ID>`.
 
 ### MySQL root user
-The root user has no password set by default. You can set it by setting
-`MYSQL_ROOT_PASSWORD` environment variable when initializing your
-database.
+The root user has no password set by default, only allowing local
+connections. You can set it by setting `MYSQL_ROOT_PASSWORD` environment
```

Comment:
Like to me it doesn't matter what would be the name, ROOT or ADMIN. The point is to stay consistent cause both of those roles, in mysql and mongo are doing the same. Why there should be a different names ?